### ISSUE TITLE
Maintenance and updates

### ISSUE THREAD

**Issue by Rastusik** [pixelfederation/coding-standards#3]
- define specific included rules for each php version from 8.1 upwards (in form of rulesets for each php version)

**Comment by pmysiak** [comment]
Yes, I already working on it. Last 40 rules have to be processed.

**Comment by pmysiak** [comment]
Already separated and one-by-one listed.
